src/openeo_gfmap/manager/job_manager.py

```diff
@@ -423,6 +530,11 @@ def run_jobs(
             self._wait_queued_actions()
             _log.info("Exiting ThreadPoolExecutor.")
             self._executor = None
+        _log.info(
+            "Finished running jobs, saving persisted STAC collection to final .json collection."
+        )
+        self._write_stac()
```

Comment:
fyi, this option will also become available in the standard multibackendjobmanager